src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/Tkn.java

```diff
@@ -166,4 +166,15 @@
      * @throws IOException
      */
     void editResource(KubernetesClient client, String namespace, String name, CustomResourceDefinitionContext crdContext, String objectAsString) throws IOException;
+
+    /**
+     * Start the execution of a task
+     *
+     * @param namespace the namespace of the task
+     * @param task the task that has to be run
+     * @param args args to start task (contains input/output params/resources)
+     * @throws IOException if communication errored
+     */
+    void startTask(String namespace, String task, List<String> args) throws IOException;
```

Comment:
This array of strings is too much linked to the CLI. I would rather prefer having an interface like:
`startTask(String namespace, String task, Map<String, String> parameters, Map<String, String> inputResources, Map<String, String> outputResources)`.
You are putting some CLI logic into the dialog (-i, -p or -o)